app/src/main/java/com/cornellappdev/android/eatery/MainActivity.java

```diff
@@ -83,32 +88,12 @@ protected void onCreate(Bundle savedInstanceState) {
     bnv = findViewById(R.id.bottom_navigation);
     splash = findViewById(R.id.relative_layout_splash);
     bnv.setVisibility(View.GONE);
+    areaButtonsPressed = new HashSet<>();
+    paymentButtonsPressed = new HashSet<>();
     if (getSupportActionBar() != null) {
       getSupportActionBar().hide();
     }
-
-    ConnectionUtilities con = new ConnectionUtilities(this);
```

Comment:
how are we checking wifi connection without this???

Reply:
It is now done in the async task